code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -684,12 +684,43 @@
 
 /datum/species/golem/clockwork/no_scrap //These golems are created through the herald's beacon and leave normal corpses on death.
 	id = "clockwork golem servant"
-	armor = 15 //Balance reasons make this armor weak
+	armor = 40
+	inherent_traits = list(TRAIT_RESISTHEAT,TRAIT_NOBREATH,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_NOFIRE,TRAIT_RADIMMUNE,TRAIT_PIERCEIMMUNE,TRAIT_NODISMEMBER,TRAIT_VIRUSIMMUNE,TRAIT_THERMAL_VISION,TRAIT_IGNOREDAMAGESLOWDOWN)
```

Comment:
Check returned: Line 6: Formatting warning: Commas should have a space after them. (3, 5, 6) not (3,5,6 or 3 ,4 ,5)